### ISSUE TITLE
how to merge multiple scalar into one figure

### ISSUE THREAD

**Issue by blueardour** [lanpa/tensorboard-pytorch#147]
Hi, thanks for the excellent library.

I have a question. When add a write.add_scalar, a new figure is generated. 
I wonder how to merge multiple scalar into one figure? It is similar with the write.add_scalars function, however I have to call the write.add_scalar in different positions of the code.

**Comment by lanpa** [comment]
Sorry I can understand your question. Doesn't `demo.py` work for your case?

**Comment by blueardour** [comment]
Yes, the demo.py work perfect for me.

I have a requirement like the following:
`
writer = SummaryWriter()
for iter in emuerate(xx):
   write.add_scalar('loss', loss, iter)
   if iter % 1000 ==0:
       write.add_scalar('top1', top1, iter)
       write.add_scalar('top5', top5, iter)
write.close()
`
After I show the log in tensorboard, the loss, top1, top5 are in different figures.
I wonder if I could merge these plot into one figure. Could it possible provide some 'write.merge()' function.

**Comment by lanpa** [comment]
What arguments would be passed to `write.merge()`?  Wouldn't it similar to `add_scalars`?

**Comment by blueardour** [comment]
Yes, I expect pass the variable name with the same x-axis(for example, all the 'loss', 'top1', 'top5' scalars employ the 'iter' as their x-axis) into the write.merge(). So that these scalars could be drawn in the same figure.
I'm sorry that I don't know  whether it is hard to implement this function.
![image](https://user-images.githubusercontent.com/4530223/40816968-664019be-6582-11e8-8942-3ac60862b34d.png)

**Comment by lanpa** [comment]
Oh, I think I got your point. Does the following code work for you?
```python
for n_iter in range(10000):
    writer.add_scalars('data/scalar_group', {'loss': n_iter*np.arctan(n_iter)}, n_iter)
    if n_iter%1000==0:
        writer.add_scalars('data/scalar_group', {'top1': n_iter*np.sin(n_iter)}, n_iter)
        writer.add_scalars('data/scalar_group', {'top5': n_iter*np.cos(n_iter)}, n_iter)
```
![image](https://user-images.githubusercontent.com/2005323/40821466-ca2f3cfe-6599-11e8-91d4-837bd4443e2e.png)



Looks like you use matplotlib, you may also interested in discussing #152 with us.

**Comment by blueardour** [comment]
Great. Thanks for the illustration. It solved my problem. I would also look at the #152 issue.

**Comment by SergiVera** [comment]
Does anybody know how to do this in Tensorflow 2.5? The aforementioned add_scalars method no longer exists.

Thanks in advance!

**Comment by pepi99** [comment]
Bump
